src/test/java/com/google/devtools/build/lib/bazel/repository/PatchUtilTest.java

```diff
@@ -404,26 +404,6 @@ public void testPatchFileNotFound() {
     assertThat(expected).hasMessageThat().contains("Cannot find patch file: /root/patchfile");
   }
 
-  @Test
-  public void testMissingBothOldAndNewFile() throws IOException {
```

Comment:
With the checking for ---/+++ lines, this error will never be thrown again, so removing this test.